converter/internal/promtailconvert/testdata/loki_write.yaml

```diff
@@ -0,0 +1,17 @@
+clients:
+  - url: http://localhost/loki/api/v1/push
+    headers:
+      fat: ball
+      bird: house
+    tenant_id: sparrow
+    batchwait: 10s
+    batchsize: 1024
+    proxy_url: http://proxy.example.com
+    backoff_config:
+      min_period: 100ms
+      max_period: 10s
+      max_retries: 20
+    drop_rate_limited_batches: true
```

Comment:
Nit: Since we're setting this to default value, maybe we should have two clients one with drop_rate_limited_batches set to true and the other set to false?